cloudevent-broker/main.tf

```diff
@@ -13,3 +13,9 @@ resource "google_pubsub_topic" "this" {
   // TODO: Tune this and/or make it configurable?
   message_retention_duration = "600s"
 }
+
+module "dashboard" {
```

Comment:
This collides with the invocation in `ingress.tf`, but I'd like to see if we can combine these into a unified broker dashboard using your building blocks 🤩